common/processes/platform/tiny_process_library/tiny_process_library.cpp

```diff
@@ -17,13 +17,29 @@
 #include <string>
 #include <string_view>
 
-#include <process.hpp>
-
-namespace tpl = TinyProcessLib;
 using namespace std;
 
 namespace mender::common::processes {
 
+error::Error Process::Start() {
+	proc_.reset(new tpl::Process(this->args_, ""));
```

Comment:
I was thinking I couldn't use `make_unique` because it is C++14, but I forgot this is a platform file. Fixed.

`this->args_` was preexisting (just moved), but I nevertheless simplified it now.